_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 ---
 layout: post
-title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
+title: "NewPipe 0.20.0 released: Major version, new unified player, new notifications. More things."
 short: "NewPipe 0.20.0 released"
 date: 2020-10-04 05:00:00 +0100
 categories: [pinned, release]
-author: Opusforlife & Poolitzer
```

Comment:
opusforlife is otherwise lowercase only, and unless the collaboration is very close, the ampersand is unwarranted. The strings don't use ampersand.

Reply:
"Can it be Poolitzer and opusforlife?" I didn't think of this before, but now that you've brought it up, I think capitalisation here alerts the reader that it is a proper noun, and hence a username.

From your link: "The Writers Guild of America uses & to denote two writers collaborating on a specific script."

We are two writers who collaborated on a specific blog post. So it's fine, then?